nservicebus/upgrades/nhibernate-saga-index-fix.md

```diff
@@ -0,0 +1,78 @@
+---
+title: NHibernate Persistence - unique constraints on saga mapping tables
```

Comment:
There is constraint on the length of article title (70 chars). Currently it's 65 :P so the suggested version will be to long.